build.gradle

```diff
@@ -81,11 +89,46 @@ jar {
     }
 }
 
-// This is a hack to disable the java 8 default javadoc lint until we fix the html formatting
 tasks.withType(Javadoc) {
+    // do this for all javadoc tasks, including gatkDoc
+    options.addStringOption('Xdoclint:none')
+}
+
+javadoc {
+    // This is a hack to disable the java 8 default javadoc lint until we fix the html formatting
+    // We only want to do this for the javadoc task, not gatkDoc
     options.addStringOption('Xdoclint:none', '-quiet')
 }
 
+// Generate Picard Online Doc
+task picardDoc(type: Javadoc, dependsOn: ['cleanPicardDoc', classes]) {
```

Comment:
@magicDGS A gradle plugin for barclay seems like a good idea.  I've never written one, but I imagine it's probably not that difficult.  I opened a ticket to discuss it #819